lib/lotus/validations/error.rb

```diff
@@ -90,6 +92,33 @@ def ==(other)
           other.expected   == expected   &&
           other.actual     == actual
       end
+
+      # FIXME for Luca - What to do when I18n isn't defined?
```

Comment:
@AlfonsoUceda Leave it to me ;) For now just assume it's available for that method :dancer: